router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -0,0 +1,46 @@
+package bingads
+
+import (
+	stdjson "encoding/json"
+
+	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/common"
+)
+
+type BING_ADS struct {
+	destName string
```

Comment:
destName is not useful

Reply:
other appropriate variables will be added